infra/gcp/monitoring/k8s.io/versions.tf

```
@@ -0,0 +1,8 @@
+/*
+This file defines:
+- Required Terraform version
+*/
+
+terraform {
+  required_version = ">= 0.13.0"
```

```suggestion
  required_version = ">= 0.14.0"
```
See: https://github.com/kubernetes/k8s.io/pull/2019
Sorry for make you revert it. :-D

---

This will need a rebase / update I think now, should be:
```suggestion
  required_version = "~> 1.0.0"
```